### ISSUE TITLE
[Content]: Braze Cloud Mode Actions destination - Debounce Middleware & Engage

### ISSUE THREAD

**Issue by lizkane222** [segmentio/segment-docs#4333]
### What article on segment.com/docs is affected?

https://segment.com/docs/connections/destinations/catalog/braze-cloud-mode-actions/#debounce-middleware

### What part(s) of the article would you like to see updated?

Per [this Slack thread](https://twilio.slack.com/archives/CC97A542H/p1677700957017759), the Debounce Middleware Action, though possible to add on the Mappings tab, is not available or applicable for destinations connected to Engage. This is because Engage hits the Tracking API directly, and thus no AJS2 to run the debounce middleware against. 

This section should say something to include that this Action is not applicable to destinations within Engage.

### Additional information

Liz Kane (CSE) lkane@twilio.com

I'm currently working with our customer (papier) who believed that the Debounce Middleware action was applicable to their Engage Braze destination. I've updated them with the above information and believe that adding this section into our docs will ensure that future customers aren't also confused by this.

**Comment by lizkane222** [comment]
Will you update the https://segment.com/docs/connections/destinations/catalog/braze-cloud-mode-actions/#debounce-middleware doc, under the Debounce Middleware section to state this : 

> Debounce functionality requires a frontend client to work. Therefore, it cannot be used with server-side libraries or with Engage.

**Comment by lizkane222** [comment]
yes the contents of my comment seems to still be missing from the doc and still needs to be updated to what I shared in my comment on Apr 27

**Comment by forstisabella** [comment]
Hi @lizkane222 - it looks like the update you made in the actions-destination repo are reflected on the docs site, so I am going to close this as complete!
